docs/source/debugging_environment_variables.rst

```diff
@@ -0,0 +1,11 @@
+.. _debugging_environment_variables:
+
+Debugging Environment Variables
+===============================
+.. list-table::
+  :header-rows: 1
+
+  * - Variable
+    - Description
+  * - ``TORCH_SHOW_CPP_STACKTRACES``
```

Comment:
I would add TORCH_LOGS here?

Reply:
So I added this at the top level docs, a link torch._logging since there is already an entire page do you think I should reference here?